### ISSUE TITLE
Broken release v2.1.0

### ISSUE THREAD

**Issue by htimur** [jamesmbourne/aws4-axios#37]
The latest release  v2.1.0 is broken due to the commit: [05f2ed46cf](https://github.com/jamesmbourne/aws4-axios/commit/05f2ed46cfbae8931d0f57476f125aff00aa6e55) not being part of the released tarball.

### How to reproduce
```
wget $(npm view aws4-axios@2.1.0 dist.tarball)   
```
And check the package/dist/interceptor.js:30
![image](https://user-images.githubusercontent.com/575813/104586987-89979e00-5666-11eb-9174-84609b404a96.png)

This leads to the error:

```
Error: Cannot find module 'axios/lib/helpers/buildUrl'
Require stack:
- .../node_modules/aws4-axios/dist/interceptor.js
```

**Comment by MoJo2600** [comment]
Can confirm

**Comment by jamesmbourne** [comment]
Hi @htimur, apologies for this  - I have released [v2.1.1](https://github.com/jamesmbourne/aws4-axios/releases/tag/v2.1.1) and added a `prepublishOnly` script to avoid this occuring in the  future.

Thanks for bringing my attention to this!

**Comment by htimur** [comment]
> Hi @htimur, apologies for this - I have released [v2.1.1](https://github.com/jamesmbourne/aws4-axios/releases/tag/v2.1.1) and added a `prepublishOnly` script to avoid this occuring in the future.
> 
> Thanks for bringing my attention to this!

Thank you for the quick fix.
